:mod:`rec_to_nwb.processing.exceptions.invalid_metadata_exception`
==================================================================

.. py:module:: rec_to_nwb.processing.exceptions.invalid_metadata_exception


Module Contents
---------------

.. py:exception:: InvalidMetadataException(*args)

   Bases: :class:`rec_to_nwb.processing.exceptions.invalid_exception.InvalidException`

   .. method:: __str__(self)




